isolate_example/lib/page_one.dart

```diff
@@ -1,33 +1,119 @@
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-//
-// http://www.apache.org/licenses/LICENSE-2.0
-//
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 
-class PerformancePage extends StatelessWidget {
+class PerformancePage extends StatefulWidget {
+  @override
+  _PerformancePageState createState() => _PerformancePageState();
+}
+
+class _PerformancePageState extends State<PerformancePage> {
+  Future<void> computeFuture = Future.value();
+
   @override
   Widget build(BuildContext context) {
     return Center(
       child: Column(
         mainAxisSize: MainAxisSize.min,
         children: <Widget>[
           SmoothAnimationWidget(),
+          Container(
+            alignment: Alignment.bottomCenter,
+            padding:
+                EdgeInsets.only(left: 0.0, top: 150.0, right: 0.0, bottom: 0.0),
+            child: Column(
+              children: <Widget>[
+                FutureBuilder<void>(
+                  future: computeFuture,
+                  builder: (context, snapshot) {
+                    return RaisedButton(
+                      child: const Text('Compute on Main'),
+                      elevation: 8.0,
+                      onPressed: (snapshot.connectionState ==
+                              ConnectionState.done)
+                          ? () {
+                              setState(
+                                () {
+                                  computeFuture = computeOnMainIsolate()
+                                    ..then(
+                                      (_) {
+                                        final snackBar1 = SnackBar(
+                                          content: Text('Main Isolate Done!'),
+                                        );
+                                        Scaffold.of(context)
+                                            .showSnackBar(snackBar1);
+                                      },
+                                    );
+                                },
+                              );
+                            }
+                          : null,
+                    );
+                  },
+                ),
+                FutureBuilder<void>(
+                  future: computeFuture,
+                  builder: (context, snapshot) {
+                    return RaisedButton(
+                      child: const Text('Compute on Secondary'),
+                      elevation: 8.0,
+                      onPressed: (snapshot.connectionState ==
+                              ConnectionState.done)
```

Comment:
Again, I would double check the formatting section of the style guide for better readability.